code/game/objects/effects/decals/decal.dm

```diff
@@ -51,32 +57,138 @@
 /obj/effect/turf_decal/stripes/asteroid/corner
 	icon_state = "ast_warn_corner"
 
+/obj/effect/turf_decal/stripes/asteroid/box
+	icon_state = "ast_warn_box"
+
+/obj/effect/turf_decal/stripes/asteroid/full
+	icon_state = "ast_warn_full"
+
+/obj/effect/turf_decal/stripes/white/line
+	icon_state = "warningline_white"
+
+/obj/effect/turf_decal/stripes/white/end
+	icon_state = "warn_end_white"
+
+/obj/effect/turf_decal/stripes/white/corner
+	icon_state = "warninglinecorner_white"
+
+/obj/effect/turf_decal/stripes/white/box
+	icon_state = "warn_box_white"
+
+/obj/effect/turf_decal/stripes/white/full
+	icon_state = "warn_full_white"
+
+/obj/effect/turf_decal/stripes/red/line
+	icon_state = "warningline_red"
+
+/obj/effect/turf_decal/stripes/red/end
+	icon_state = "warn_end_red"
+
+/obj/effect/turf_decal/stripes/red/corner
+	icon_state = "warninglinecorner_red"
+
+/obj/effect/turf_decal/stripes/red/box
+	icon_state = "warn_box_red"
+
+/obj/effect/turf_decal/stripes/red/full
+	icon_state = "warn_full_red"
+
 /obj/effect/turf_decal/delivery
 	icon_state = "delivery"
 
+/obj/effect/turf_decal/delivery/white
+	icon_state = "delivery_white"
+
+/obj/effect/turf_decal/delivery/red
+	icon_state = "delivery_red"
+
 /obj/effect/turf_decal/bot
 	icon_state = "bot"
 
+/obj/effect/turf_decal/bot/right
+	icon_state = "bot_right"
+
+/obj/effect/turf_decal/bot/left
+	icon_state = "bot_left"
+
+/obj/effect/turf_decal/bot_white
+	icon_state = "bot_white"
+
+/obj/effect/turf_decal/bot_white/right
+	icon_state = "bot_right_white"
+
+/obj/effect/turf_decal/bot_white/left
+	icon_state = "bot_left_white"
+
+/obj/effect/turf_decal/bot_red
+	icon_state = "bot_red"
+
+/obj/effect/turf_decal/bot_red/right
+	icon_state = "bot_right_red"
+
+/obj/effect/turf_decal/bot_red/left
+	icon_state = "bot_left_red"
+
 /obj/effect/turf_decal/loading_area
-	icon_state = "loading_area"
+	icon_state = "loadingarea"
 
-/obj/effect/turf_decal/sand

```

Comment:
Why are these paths being removed? They're warning overlays for sandy/beach areas.

Reply:
Because we already have both sandy transparent overlays for a full regular tile/plating AND sandy warning lines (which are already pre-made results of placing those overlays on them). So those were redundant and also lacking in options (end, box, full).